src/routes/api.php

```diff
@@ -17,3 +17,7 @@
 Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
```

Comment:
Waarom duplicate?

Reply:
.api mag sws helemaal leeg. Gaan we niet gebruiken